### PR TITLE
Nfdiv 3787 switch to sole

### PR DESCRIPTION
**Pull request by velimd**
### Change description ###

Enter a description.

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/NFDIV-3787

### Pull request checklist ###

**Before raising**
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] README and other documentation has been updated / added (if needed)

**Before merging**
- [ ] this ticket been reviewed by QA
- [ ] the user story been signed off by the PO

**Note:** Bug fixes, dependency updates and technical tasks do not directly impact the user experience and can be merged without QA and PO review.

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/divorce/common/event/SwitchedToSoleCo.java [hunk 1]
```diff
@@ -115,51 +93,30 @@ public AboutToStartOrSubmitResponse<CaseData, State> aboutToSubmit(CaseDetails<C
...
         notificationDispatcher.send(switchToSoleCoNotification, data, details.getId());
 
         if (CO_D84.equals(data.getDocuments().getTypeOfDocumentAttached())
-                || D84.equals(data.getDocuments().getScannedSubtypeReceived())
+            || D84.equals(data.getDocuments().getScannedSubtypeReceived())
             && SWITCH_TO_SOLE.equals(data.getConditionalOrder().getD84ApplicationType())) {
 
-            switchToSoleCoPrinter.print(data, details.getId(), data.getApplicant2());
+            var documentPackInfo =
+                switchToSoleConditionalOrderDocumentPack.getDocumentPack(data, null);
```
Comment (ed14537) [inline src/main/java/uk/gov/hmcts/divorce/common/event/SwitchedToSoleCo.java:107]: Why isn't this call in the notification class?

Comment (velimd) [reply src/main/java/uk/gov/hmcts/divorce/common/event/SwitchedToSoleCo.java:107]: I can move it but as the original was placed here, I left it here.

Comment (ed14537) [reply src/main/java/uk/gov/hmcts/divorce/common/event/SwitchedToSoleCo.java:107]: oh right as in it would send an email AND send to bulk print?

Comment (velimd) [reply src/main/java/uk/gov/hmcts/divorce/common/event/SwitchedToSoleCo.java:107]: kinda, itll send the email yh but the bulk print will depend on the condition as its in an if statement.

Comment (ed14537) [reply src/main/java/uk/gov/hmcts/divorce/common/event/SwitchedToSoleCo.java:107]: cool well as long as it works we can leave as is

**Review by ed14537 — APPROVED** [review]

**Pull request merged**